config/CLIConfiguration.ts

```diff
@@ -258,8 +258,17 @@ class CLIConfiguration {
 
   getConfigForAccount(accountId?: number): CLIAccount_NEW | null {
     if (this.config) {
-      this.config.accounts.find(account => account.accountId === accountId) ||
-        null;
+      return (
+        this.config.accounts.find(account => account.accountId === accountId) ||
+        null
+      );
+    }
+    return null;
+  }
+
+  getConfigAccounts(): Array<CLIAccount_NEW> | null {
```

Comment:
This can just be `getAccounts()`, which is more inline with the existing `getAccount()` function above

Reply:
I think we could do the same rename for `getConfigAccountIndex()` --> `getAccountIndex()`. These are semi-nits, but I think the consistency will help later on